Lesson02/cardDeck.js

```diff
@@ -0,0 +1,9 @@
+const suits = new Array('Heart', 'Diamond', 'Spade', 'Club');
+const ranks = new Array('Ace', '2', '3', '4', '5', '6', '7', '8', '9', '10', 'Jack', 'Queen', 'King');
```

Comment:
An easier way to declare these would be

`const suits = ['hearts', 'diamond']`

We don't need the new keyword, although it is a nice touch. But simpler is better